go.sum

```diff
@@ -431,6 +431,7 @@ github.com/modern-go/concurrent v0.0.0-20180306012644-bacd9c7ef1dd/go.mod h1:6dJ
 github.com/modern-go/reflect2 v1.0.1 h1:9f412s+6RmYXLWZSEzVVgPGK7C2PphHj5RJrvfx9AWI=
 github.com/modern-go/reflect2 v1.0.1/go.mod h1:bx2lNnkwVCuqBIxFjflWJWanXIb3RllmbCylyMrvgv0=
 github.com/mwitkow/go-conntrack v0.0.0-20161129095857-cc309e4a2223/go.mod h1:qRWi+5nqEBWmkhHvq77mSJWrCKwh8bxhgT7d/eI7P4U=
+github.com/nicksnyder/go-i18n v2.0.3+incompatible h1:XCCaWsCoy4KlWkhOr+63dkv6oJmitJ573uJqDBAiFiQ=
```

Comment:
@drmmarsunited do you know where this sum came from.  We are trying to track it down and can't figure it out.  The go.mod just has package removals and comment changes to the `/indirect` which doesn't seem like it should add something.

Reply:
No, I saw that as well.  I didn't do anything special to pull in the code, just ran a `go get`.  I can try to refork and do it from scratch and see if it comes up again?

Reply:
we are trying a few things here too but haven't been able to track it down.

Reply:
Let me try cloning to a whole new directory and populating the modules.  I'll let you know if I see the sum change or not.

Reply:
@kddejong So if I clone the master branch of my fork (which is unchanged from upstream), and I run `go get` in the newly cloned folder, the sum changes with this same line.